README.md

```diff
@@ -198,6 +206,26 @@ A numeric value that can be parsed by [`(float)` cast, equivalent to `floatval`]
 ###### `bool` default
 `true`, `on`, `yes` or `1` will result in true. Other values, as long as there is something, will result default false. (If there is nothing, the variable will not be optional)
 
+###### `timestamp` default
+A numeric value that can be parsed by [`(int)` cast, equivalent to `intval`](https://php.net/intval) or [`(float)` cast, equivalent to `floatval`](https://php.net/floatval)
+
+**Allowed default values:**
```

Comment:
This font doesn't look good in rendered form. 
![image](https://user-images.githubusercontent.com/19623715/85128858-af8dc900-b264-11ea-80b5-dc95775ab7ec.png)